src/components/LoanCards/ListLoanCard.vue

```diff
@@ -66,7 +66,7 @@
 						@track-loan-card-interaction="trackInteraction"
 					/>
 				</div>
-				<div class="large-7 columns show-for-large">
+				<div class="large-7 columns show-for-large tw-py-1">
```

Comment:
Does this 'double" the padding at larger + smaller screen sizes?

Reply:
Maybe not if there is a margin collapse situation occurring. Just checking as the small and xl versions seem to already have the intended padding around the elements.

Reply:
This element didn't have top/bottom padding before, so it works out. First screenshot is large, second is small. Small actually looks a bit better now, since the top of the green box was pretty close to the element above it. The extra whitespace below in the small example is there regardless of this new padding.

![image](https://user-images.githubusercontent.com/16867161/177892651-b5d156eb-f398-4b6c-9c35-d594f27850bd.png)

![image](https://user-images.githubusercontent.com/16867161/177892705-93aad795-c4a4-49bd-bd5b-7cfc5bff9e16.png)

Reply:
This padding is also applied to the lend buttons, since it's the same element wrapper, but again, the changes actually look a bit better in small view:

![image](https://user-images.githubusercontent.com/16867161/177892828-ec7afb23-730b-45c4-9138-c43c4c86fd6d.png)